workers/common/src/main/kotlin/common/env/definition/EnvironmentVariableDefinition.kt

```diff
@@ -21,16 +21,28 @@ package org.eclipse.apoapsis.ortserver.workers.common.env.definition
 
 import org.eclipse.apoapsis.ortserver.model.Secret
 
+interface EnvironmentVariableDefinition {
```

Comment:
Please update documentation. The interface and the implementing classes should have correct documentation.
Also, this should probably be a `sealed` interface.